Azure/sonic-buildimage
# Create DaemonBase class for all daemons

This new python wheel is to wrap the common function of daemons,
such as logging, signal handling, load specific platform plugins.

Signed-off-by: Kevin Wang <kevinw@mellanox.com>



**- What I did**
 Create a new python wheel named SONIC_DAEMON_COMMON_PY2 to wrap the common functions/macros for daemons. 
**- How I did it**

**- How to verify it**
Change the daemons(ledd,xcvrd,psud) to import and use this common wheel. The daemons should work normally.
**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
